### ISSUE TITLE
Creating a form by opening a form file does not save it to account when signed in

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#3628]
## Description
When importing a JSON form file to create a form, it opens in the Preview/Test section. If you go to Forms, that form does not save to your forms. It seems you need to take the extra step to go to Edit and actually name the file and click save? This could lead to confusion and losing forms when importing more than one in succession.

## Steps to reproduce
1. Go to GC Forms
2. Open a form file
3. Go to Forms
4. Go to Create new form
5. Open another form file
6. Go to Forms
7. Notice it's empty

## Details
- **Browser**: Edge
- **Operating system**: Windows
- **Environment**: Staging
- **Language**: French
- **Viewport**: Desktop

## Expected behaviour
Auto-named and auto-saved when imported?

## Screenshots or videos

![Opens to preview but does not save](https://github.com/cds-snc/platform-forms-client/assets/38330843/2cfc918f-088c-411e-98ae-930eef1af671)

**Comment by timarney** [comment]
Been wondering for a while if we should be taking folks to edit vs preview first ... thoughts?

**Comment by anikbrazeau** [comment]
Yeah I think that might be more expected? Or I wonder if conserving the name of the file and saving it when uploaded might also help if they land on preview?

**Comment by Abi-Nada** [comment]
Hi folks! This behaviour currently persists in Production - I'm going to de-scope it from the App Router work, and we can prioritize outside of this work

**Comment by Abi-Nada** [comment]
Current functionality is resulting in a few errors/support tickets when a user uploads a JSON file and immediately updates Settings before saving a draft. (These issues are labelled as "blocking" within this card).

**Comment by anikbrazeau** [comment]
This issue #4111 came up again in a recent support ticket: 
https://gcdigital.slack.com/archives/C03T4FAE9FV/p1729177281868809?thread_ts=1729103502.367649&cid=C03T4FAE9FV

**Comment by anikbrazeau** [comment]
This should be resolved by https://github.com/cds-snc/platform-forms-client/pull/5000
